class-astra-theme-customizer-reset.php

```diff
@@ -125,22 +125,29 @@ public function ajax_customizer_reset() {
 		public function controls_scripts() {
 
 			// Enqueue JS.
-			wp_enqueue_script( 'astra-theme-customizer-reset', ASTRA_THEME_CUSTOMIZER_RESET_URI . 'assets/js/customizer-reset.js', array( 'jquery', 'astra-customizer-controls-toggle-js' ), null, true );
+			wp_enqueue_script( 'astra-theme-customizer-reset', ASTRA_THEME_CUSTOMIZER_RESET_URI . 'assets/js/customizer-reset.js', array( 'jquery', 'astra-customizer-controls-toggle-js' ), true, true );
 
 			// Add localize JS.
-			wp_localize_script( 'astra-theme-customizer-reset', 'astraThemeCustomizerReset', apply_filters( 'astra_theme_customizer_reset_js_localize', array(
-				'customizer' => array(
-					'reset' => array(
-						'stringConfirm' => __( 'Warning! This will remove all the Astra theme customizer options!', 'reset-astra-customizer' ),
-						'stringReset'   => __( 'Reset All', 'reset-astra-customizer' ),
-						'nonce'         => wp_create_nonce( 'astra-theme-customizer-reset' ),
-					),
-				),
-			) ) );
+			wp_localize_script(
+				'astra-theme-customizer-reset',
+				'astraThemeCustomizerReset',
+				apply_filters(
+					'astra_theme_customizer_reset_js_localize',
+					array(
+						'customizer' => array(
+							'reset' => array(
+								'stringConfirm' => __( 'Warning! This will remove all the Astra theme customizer options!', 'astra-customizer-reset' ),
+								'stringReset'   => __( 'Reset All', 'astra-customizer-reset' ),
+								'nonce'         => wp_create_nonce( 'astra-theme-customizer-reset' ),
```

Comment:
@shubhw12 Please verify the nonce here.

Reply:
Done

Reply:
It is verified in the customizer-reset.js